.github/workflows/block-fixup-commits.yaml

```diff
@@ -0,0 +1,30 @@
+name: Block "fixup" commit merges
+
+on:
+  workflow_call:
+
+jobs:
+  check-commits:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+      with:
+        ref: ${{ github.event.pull_request.head.ref }}
```

Comment:
```suggestion
        ref: ${{ github.event.pull_request.head.ref }}  # To ensure we have an attached HEAD
```

Reply:
Fixed in the [forcepush](https://github.com/python-discord/.github/commit/f2f1f2db0b1dcb2f44f0a619d384c9118e5be983)